apps/website/.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+	"typescript.tsdk": "../../node_modules/typescript/lib",
```

Comment:
Git ignore this file. The path is os specific and everytime a user uses the build command on windows, it'll change. I copied these 2 lines in the root of my monorepo and gitignored this file.